app/models/user.rb

```diff
@@ -19,6 +19,12 @@ class User < ActiveRecord::Base
   has_many :courses, through: :course_users
   has_many :course_group_users, dependent: :destroy, class_name: Course::GroupUser.name
   has_many :course_groups, through: :course_group_users, class_name: Course::Group.name
+  has_many :posts, class_name: Discussion::Post.name, inverse_of: :author, foreign_key: :author_id
+  has_many :topics, class_name: Discussion::Topic.name, inverse_of: :author, foreign_key: :author_id
```

Comment:
Maybe we can have some fields like "Attended Topics" and "My Comments" in user's dash board in the future?